src/main/java/mate/academy/springboot/datajpa/service/GeneralService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.springboot.datajpa.service;
+
+public interface GeneralService<T> {
+    T create(T model);
```

Comment:
let's rename this method for better matching what this method do and better code readability
```suggestion
    T save(T model);
```